llama_stack/providers/utils/memory/openai_vector_store_mixin.py

```diff
@@ -147,8 +147,9 @@ async def openai_create_vector_store(
         provider_vector_db_id: str | None = None,
     ) -> VectorStoreObject:
         """Creates a vector store."""
-        store_id = name or str(uuid.uuid4())
         created_at = int(time.time())
+        if provider_vector_db_id is None:
```

Comment:
This will adjust to be an unconditional setting of this now, I guess?

Reply:
Yup!